src/bmp/mod.rs

```diff
@@ -8,7 +8,8 @@
 //!
 
 pub use self::decoder::BmpDecoder;
-pub use self::encoder::BMPEncoder;
+#[allow(deprecated)]
+pub use self::encoder::{BmpEncoder, BMPEncoder};
```

Comment:
Since you commented on this in the description, I think it is perfectly fine to have the attribute ourselves currently.